src/Cake.Issues/Issue.cs

```diff
@@ -64,7 +66,8 @@ public class Issue : IIssue
             Uri ruleUrl,
             string run,
             string providerType,
-            string providerName)
+            string providerName, 
```

Comment:
There's a StyleCop warning here:

- SA1028: Code should not contain trailing whitespace